Warsaw_Tram/TramViewModel.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  TramViewModel.swift
+//  Warsaw_Tram
+//
+//  Created by Małgorzata Dziubich on 28/06/16.
+//  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
+//
+
+import UIKit
+
+protocol TramViewModelProtocol {
+    
+    func getTramsData(success: [Tram] -> Void, failure: String -> Void) -> Void
+}
+
+class TramViewModel: TramViewModelProtocol {
+    
+    let resourceId = "id=c7238cfe-8b1f-4c38-bb4a-de386db7e776"
+    let warsawTramsApiKey = "apikey=060b903c-b0c3-427e-934d-9e4a81a61969"
+
+    func getTramsData(success: [Tram] -> Void, failure: String -> Void) -> Void {
+        var trams = [Tram]()
+        let stringURL = "https://api.um.warszawa.pl/api/action/wsstore_get?\(resourceId)&\(warsawTramsApiKey)"
+        let url = NSURL(string: stringURL)!
+        
+        let request = NSMutableURLRequest(URL: url)
+        request.HTTPMethod = "GET"
+        
+        let task = NSURLSession.sharedSession().dataTaskWithRequest(request) { data, response, error in
+            
+            if error != nil {
+                failure(error!.localizedDescription)
+            } else if data != nil {
+                guard let jsonResult = (try! NSJSONSerialization.JSONObjectWithData(data!, options: NSJSONReadingOptions.MutableContainers)) as? NSDictionary,
+                    let result: NSArray = jsonResult["result"] as? NSArray else {
```

Comment:
I just got an error when using Array, but now I changed it and it looks like this:
`let result: Array = jsonResult["result"] as? Array<AnyObject>`